src/SharpGLTF.Toolkit/Geometry/VertexTypes/VertexMaterial.cs

```diff
@@ -128,6 +142,24 @@ public static bool AreEqual(in VertexColor1 a, in VertexColor1 b)
 
         #region API
 
+        /// <inheritdoc/>
+        public VertexMaterialDelta Subtract(IVertexMaterial baseValue)
+        {
+            return new VertexMaterialDelta((VertexColor1)baseValue, this);
+        }
+
+        /// <inheritdoc cref="Subtract(IVertexMaterial)"/>
+        public VertexMaterialDelta Subtract(in VertexColor1 baseValue)

```

Comment:
As a general case I try to avoid adding more API surface unless it's really useful. The "add-subtract" APIs in the vertices it's something I added as a neccesary evil and not something I really wanted to do. Also, the more methods we add to the API, the more methods need to be maintained in the long run to avoid breaking compatibility, also more methods to document, etc. Personally speaking,

Personally, I would have preferred to make these methods internal, but at some point I thought, well, it might be useful to someone.

I really don't know what to do; if these methods is something users might use extensively, certainly having a method without boxing/unboxing can improve performance.... but how many developers out there will actually use them? and it's worth for all other developers to make the API more complex? (that's the kind of thoughts I consider when adding an api)

I leave it to you, whether you want to add the rest of the methods, or remove them